### ISSUE TITLE
TTY not available when debugging in GDB mode

### ISSUE THREAD

**Issue by TimKendall76** [Microsoft/VSLinux#33]
Please use the following bug reporting template to help produce actionable and reproducible issues:

* A brief description

* Expected results

* Actual results (with build output window contents if applicable)

* VC++ version, Linux system name and version, GCC/GDB/gdbserver version, CPU arch, etc. If you are using a specific version of Linux on an embedded board, we might need to know about it to get a repro on similar hardware

* Steps required to reproduce the error

* Required packages and commands to install

See [our contributing instructions](/CONTRIBUTING.md) for assistance.

**Comment by TimKendall76** [comment]
Developing a SO library and need to use GDB mode to debug. STDOUT goes to the Output window and STDIN don't no where, neither are attached to the Linux Console window under "Debug" in VS. If a getchar is issued, debugging actually stops dead. Apparently when in GDB mode, there is only one SSH connection whereby two are needed to provide console I/O. Workaround is to statically link library to debug and use gdbserver mode. Thanks to you Stan.
VC_Linux 1.0.5

**Comment by robotdad** [comment]
Yes the console window isn't wired up for gdb mode today, this is something we need to look at.

**Comment by robotdad** [comment]
We're finally pulling this one up onto the backlog.
